packages/hooks/src/get-hooks.js

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env node
+// eslint-disable-next-line no-console
+console.log(JSON.stringify({
+  hooks: {
+    'get-manifest': 'npx -q --no-install -p @slack/hooks slack-cli-get-manifest',
+    'check-update': 'npx -q --no-install -p @slack/hooks slack-cli-check-update',
+    start: 'npx -q --no-install -p @slack/hooks slack-cli-start',
+  },
+  config: {
+    watch: {
+      'filter-regex': '^manifest\\.(ts|js|json)$',
```

Comment:
Have we confirmed that `.ts` and `.js` work as expected? Per [`get-manifest.js`](https://github.com/slackapi/node-slack-sdk/pull/1714/files#diff-be817d63e5b2302e9b335351016f0a0c2de6c5488527691df45e45175df73b6bR21), I believe we only support `manifest.json` atm.

Reply:
Totally right that `.ts` and `.js` isn't supported right now. I'll remove these!